samples/Basic_ProgMem/app/TestProgmem.cpp

```diff
@@ -131,6 +131,23 @@ void testFSTR(Print& out)
 	TEST(demoFSTR1 == String(demoFSTR2))
 #undef TEST
 
+
+	// FSTR table
+
+	static DEFINE_FSTR(fstr1, "Test string #1");
+	static DEFINE_FSTR(fstr2, "Test string #2");
+
+	static FSTR_TABLE(table) = {
+			FSTR_PTR(fstr1),
+			FSTR_PTR(fstr2),
+	};
+
+	// Table entries may be accessed directly as they are word-aligned
```

Comment:
Make sure to run `make cs` and commit again.

```
!!! samples/Basic_ProgMem/app/TestProgmem.cpp not compliant to coding style, here is the fix:
--- samples/Basic_ProgMem/app/TestProgmem.cpp	2018-10-05 08:17:28.813803129 +0000
+++ /dev/fd/63	2018-10-05 08:17:34.313872466 +0000
@@ -131,15 +131,14 @@
 	TEST(demoFSTR1 == String(demoFSTR2))
 #undef TEST
 
-
 	// FSTR table
 
 	static DEFINE_FSTR(fstr1, "Test string #1");
 	static DEFINE_FSTR(fstr2, "Test string #2");
 
 	static FSTR_TABLE(table) = {
-			FSTR_PTR(fstr1),
-			FSTR_PTR(fstr2),
+		FSTR_PTR(fstr1),
+		FSTR_PTR(fstr2),
 	};
 
 	// Table entries may be accessed directly as they are word-aligned
```

Reply:
Weird. The travis run on my repo didn't pick this up!

Reply:
I set travis up on my github account so it could pre-check the commit before submitting it. Same file, here's the result:
```The command "if [ "$SDK_VERSION" == "2.0.0" ]; then ../.travis/tools/clang/format-pr.sh; fi" exited with 0.```

Not sure how to go about debugging that - it shouldn't need any additional settings to get it working?

Reply:
Just run locally the following commands

```
cd %SMING_HOME%
make cs # or in the worst cases open the file with very basic text editor and fix the indentation.
git add -u 
git commit -m "Fixed coding style issues"
git push
```

Reply:
Yep, I need to remember to run ```make cs``` ! Thanks.